plat/rockchip/rk3399/drivers/dram/dfs.c

```diff
@@ -445,7 +445,7 @@ static uint32_t get_pi_tdfi_phy_rdlat(struct dram_timing_t *pdram_timing,
 	} else if (timing_config->dram_type == LPDDR3) {
 		mem_delay_ps = 5500;
 	} else {
-		printf("get_pi_tdfi_phy_rdlat:dramtype unsupport\n");
+		ERROR("get_pi_tdfi_phy_rdlat:dramtype unsupport\n");
```

Comment:
Minor comment - in general ERROR is used where the next step is to panic(), call plat_error_handler(), or the function is part of an assert.
This doesn't seem to be the case here - so would NOTICE be more appropriate?